tests/euler.cc

```diff
@@ -50,9 +50,13 @@ TEST(CPU, TEMPORAL_SCHEME_EULER)
   GALS::CPU::Array<T_GRID, T> alpha(grid);
   GALS::CPU::Array<T_GRID, T> convection(grid);
   GALS::CPU::Array<T_GRID, T> alpha_new(grid);
+  GALS::CPU::Levelset<T_GRID, T> levelset(grid);
+
+  levelset.phi() = alpha_new;
```

Comment:
Haven't you changed the use of alpha variable to levelset?

Reply:
Yes I ignored this file, since we need semi-lagrangian schemes. I will fix it.

Reply:
fixed.